src/ArmTemplates/ServiceExtensions.cs

```diff
@@ -54,6 +56,13 @@ public static void AddArmTemplatesServices(this IServiceCollection services, ILo
             SetupBuilders(services);
             SetupExtractors(services);
             SetupCreators(services);
+            SetupDataProcessors(services);
+        }
+
+        static void SetupDataProcessors(IServiceCollection services) {
```

Comment:
please, move `{` to a new line -> [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions)

Reply:
Updated, thank you for catching this!